hello.md

```diff
@@ -1,6 +1,8 @@
-# hello
-My name is Cow.
+#hello
```

Comment:
The hashtag and space is meant to indicate "hello" is of format heading 1 fyi. The file type is markdown.